File: src/main/java/dev/sky_lock/pocketlifevehicle/command/ICommand.java
package dev.sky_lock.pocketlifevehicle.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

/**
 * @author sky_lock
 */

public interface ICommand {

    void execute(CommandSender sender, Command cmd, String[] args);
}

File: src/main/kotlin/dev/sky_lock/pocketlifevehicle/command/lib/ICommand.kt
package dev.sky_lock.pocketlifevehicle.command.lib

import org.bukkit.command.Command
import org.bukkit.command.CommandSender

/**
 * @author sky_lock
 */

interface ICommand {

    fun run(sender: CommandSender, command: Command, label: String, args: Array<String>): Int
}